### .ufo/waves/round-48/receipts/xkbcli-compile-keymap-r48.txt
old_sha=ab5fa9716930 new_sha=83c07d33ca3f old_len=14270 new_len=14270
--- a/.ufo/waves/round-48/receipts/xkbcli-compile-keymap-r48.txt
+++ b/.ufo/waves/round-48/receipts/xkbcli-compile-keymap-r48.txt
@@ -161,5 +161,5 @@
 $ /usr/bin/xkbcli compile-keymap --test --layout br --options nope:nope
 xkbcommon: ERROR: [XKB-595] Unrecognized RMLVO option "nope:nope" was ignored
-rc=0
+rc=1
 
 --- unknown model, KcCGST

